### PR TITLE
Comparison page improvements

### PR DESCRIPTION
**Pull request by ericbieller**

### CODE REVIEW

File: comparison/speak-vs-skype/index.html [hunk 1]
```diff
@@ -8,70 +8,82 @@
...
+        <img src="/images/comparison/integrated.png" class="full-img" />
+      </p>
+    </div>
+  </div>
+
+  <div class="bar testimonial">
+    <div class="container">
+      <div class="fourteen offset-by-two columns">
+        <div class="wrapper">
+          <span>
```
Comment (tommoor) [inline comparison/speak-vs-skype/index.html]: y u lose the semantic blockquote? :cry:

Comment (ericbieller) [inline comparison/speak-vs-skype/index.html]: was commented out, went ahead and replaced with proper element though

**Comment by tommoor** [comment]
Looks like a generated directory got commited `.asset_pipeline`

**Comment by ericbieller** [comment]
@tommoor sorted

**Comment by tommoor** [comment]
Sweet, merging.

**Pull request merged**